src/test/scala/com/github/mrpowers/spark/fast/tests/DataFrameComparerTest.scala

```diff
@@ -178,12 +175,9 @@ class DataFrameComparerTest extends FreeSpec with DataFrameComparer with SparkSe
         )
       )
 
-      val e = intercept[DatasetSchemaMismatch] {
+      val _ = intercept[DatasetSchemaMismatch] {
```

Comment:
If you do not want the result value, you can delete the assignment.

Reply:
Updated